tutor/src/models/task-plans/student/task.js

```diff
@@ -93,12 +98,22 @@ class StudentTask extends BaseModel {
     return this.completed_steps_count > 0;
   }
 
+  @computed get isTeacherStudent() {
+    return true === get(this, 'tasks.course.currentRole.isTeacherStudent');
+  }
+
+  @computed get isOpen() {
+    return this.opens_at < Time.now;
+  }
+
   @computed get canWork() {
```

Comment:
`canWork()` is misleading... I think this is more like... `canView()` or `canViewSteps()`
If the task is deleted they can't work but they can sometimes view the steps.

Reply:
yeah,  that's true but I'll leave it alone for now since this is used several different spots